tests/test_bad_fonts_nonrecursive.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
You can pass environment variables, see `AM_TESTS_ENVIRONMENT` in `Makefile.am`.